src/main/java/com/simanavets/tenderflexbackend/entity/User.java

```diff
@@ -0,0 +1,33 @@
+package com.simanavets.tenderflexbackend.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@Table(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String username;
+
+    private String password;
+
+    private String email;
+
+    @ManyToOne
```

Comment:
If we use ManyToOne for Roles it means that the user can have one Role, meanwhile it also means  that one Role can have connection to many Users. so you need add field "Set<User> users" to Role and add OneToMany  annotation